vars/uiBuild.groovy

```diff
@@ -78,3 +114,9 @@ static String getModuleVersion(String moduleName, boolean releaseVersion = false
     throw new RuntimeException("Unable to get ${moduleName} version. Url: ${registry.getURL()}. Status code: ${registry.getResponseCode()}.")
   }
 }
+
+@NonCPS
+static def make_tpl(String tpl, Map data) {
```

Comment:
Not ideal solution, please mark it as method that should be refactored

Reply:
Thanks Boss, marked it.